lib/coupon_code.rb

```diff
@@ -1,4 +1,4 @@
-require "coupon_code/version"
+require 'coupon_code/version'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.